mobile/src/screens/home/BirthDataScreen.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
```

Comment:
Should load existing data when opened.

Test that new line characters in textarea is saved/loaded correctly